libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts

```diff
@@ -228,10 +230,10 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
   #_triggerButton: ElementRef | undefined;
 
   constructor() {
-    this.labelDefault = this.#defaultInputProvider?.getValue<string>(
-      'dropdown',
-      'label'
-    );
+    this.#contentInfoProvider?.getContentInfo().subscribe((info) => {
```

Comment:
This is never unsubscribed. While it does create two subscriptions based on the logic there, I'd opt for using the `async` pipe in the template.